cfme/infrastructure/virtual_machines.py

```diff
@@ -336,7 +336,7 @@ class InfraVmSnapshotAddView(InfraVmView):
     @property
     def is_displayed(self):
         """Is this view being displayed"""
-        return False
```

Comment:
**REQUIRED**: Form views we generally always say False on to ensure a redirection in the case that the form was already half filled. @quarckster @jkrocil Please weigh in here